phase-5/submissions/hashtags/sundafa

```diff
@@ -0,0 +1 @@
+desmosvaloper1r7p3c6rnqdw69np77kj5k22zffyauq7ly2l5dj
```

Comment:
Hi @sundafa! I think You've written here the wrong value. This should be a transaction hashtag! Let me know when you update this so that I can merge this PR :wink:

Reply:
Ok, @RiccardoM I have changed the correct hash, now it is correct.

Reply:
Hi @sundafa, I'm sorry but the transaction hash you provided does not represent a post with an hashtag inside its message. Do you mind performing again this step of the Phase please? https://primer.desmos.network/phase-5/hashtag-post.html

Reply:
I executed a new command, but I don’t know if the message conforms to the format, I just need to add "#info" to the message?

Reply:
@sundafa yes, that's great!